README.md

```diff
@@ -174,8 +174,10 @@ For example, you may want to configure some JSON schemas before editor mounted,
 class App extends React.Component {
     editorWillMount(monaco) {
         monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
+            validate: true,
             schemas: [{
                 uri: "http://myserver/foo-schema.json",
+                fileMatch: ['*'],
```

Comment:
I don't think this line is necessary. See https://github.com/vega/editor/blob/36abaaf7967920ef90950b9133e586c95934b28d/src/components/input-panel/spec-editor/renderer.tsx

Reply:
Yes, correct. I see that if `$schema` property is specified into json, `fileMatch` is not necessary.

Meanwhile, it is necessary if `$schema` property is not given.

Reply:
Ahh, thanks for the explanation.